android/engine/src/main/java/org/smartregister/fhircore/engine/data/local/register/RegisterRepository.kt

```diff
@@ -328,7 +328,9 @@ constructor(
       paramsList
         ?.filter {
           (it.paramType == ActionParameterType.PARAMDATA ||
-            it.paramType == ActionParameterType.UPDATE_DATE_ON_EDIT) && it.value.isNotEmpty()
+            it.paramType == ActionParameterType.UPDATE_DATE_ON_EDIT) &&
+            it.value.isNotEmpty() &&
+            it.isFilterParam == true
```

Comment:
Why do we need this new property? can we have a list of all the param types (instead) that at filterable and confirm if the current `paramType` is in the list.